src/hopen/renderer/env.cljc

```diff
@@ -99,4 +99,8 @@
     'join  str/join
     'cap   str/capitalize
     'upper str/upper-case
-    'lower str/lower-case}})
+    'lower str/lower-case
+    'escape-html util/escape-html
+
+    'empty?  empty?
+    'not     not}})
```

Comment:
Hmm... I don't quite understand how you propose to carry on with development in this project. 

In the course of developing the parser, I noticed that a function is needed, so I added it in this isolated branch. No one else is impacted by the change.

These are functions that many parser are likely to need, so it makes sense to add it to the renderer itself, making it available to all parsers.

The nature of a pull request is to propose and request code changes. So this is a valid channel for making requests known. An implementation is also already provided.

By adding the functions that I need, the moment they are needed, I could continue to carry on with development without having to wait for other people. If any changes were found to be objectionable, they could be rolled back or changed. The benefit here is that the rest of the work that depends on that change will have been fleshed out and working, instead of being stopped.

To me, all of the above sounds reasonable. What's the objection here?

Reply:
There is nothing wrong about adding some code in our branch to make it work.

The problem arises when you ask to merge it. If you want your code to be easy to be discussed, maybe amended and merged, a separated PR would be better.

Reply:
We're already discussing the code here? If you want something amended, you can just leave a comment as usual?